tests/cms/views/poi/test_poi_form.py

```diff
@@ -30,3 +56,162 @@ def test_barrier_free_and_organization_box_appear(
 
     assert organization_box in response.content.decode("utf-8")
     assert barrier_free_box in response.content.decode("utf-8")
+
+
+# Choose a region
+REGION_SLUG = "augsburg"
+
+
+def create_used_poi(region_slug: str) -> int:
+    """
+    A helper function to create a new POI used in an event
+    """
+    region = Region.objects.filter(slug=region_slug).first()
+    event = Event.objects.filter(region=region).first()
+    poi_category = POICategory.objects.first()
+
+    used_poi = POI.objects.create(
+        region_id=region.id,
+        address="Adress 42",
+        postcode="00000",
+        city="Augsburg",
+        country="Deutschland",
+        latitude="48.3780446",
+        longitude="10.8879783",
+        category=poi_category,
+    )
+
+    german_language = Language.objects.filter(slug="de").first()
+    POITranslation.objects.create(
+        title="Ort",
+        slug="ort",
+        status=status.PUBLIC,
+        content="",
+        language=german_language,
+        poi=used_poi,
+    )
+
+    event.location = used_poi
+    event.save()
+
+    assert used_poi.events.count() > 0
+
+    return used_poi.id
+
+
+@pytest.mark.django_db
+def test_poi_in_use_not_archived(
```

Comment:
```suggestion
def test_poi_in_use_cannot_be_archived(
```
Nit: I prefer this wording as I find it a bit more self-explanatory